rslib/src/scheduler/fsrs/memory_state.rs

```diff
@@ -43,14 +44,18 @@ impl Collection {
     /// memory state should be removed.
     pub(crate) fn update_memory_state(
         &mut self,
-        entries: Vec<(Option<UpdateMemoryStateRequest>, SearchNode)>,
+        entries: Vec<(
+            Option<UpdateMemoryStateRequest>,
+            SearchNode,
+            TimestampMillis,
+        )>,
```

Comment:
It would be a bit cleaner to define a struct for this, as tuples get unwieldy as they grow in size, and that will let us label the TimestampMillis value.